README.md

```diff
@@ -20,56 +29,50 @@ Your site is now running at `http://localhost:8000`!
 
 Note: You'll also see a second link: `http://localhost:8000/___graphql`. This is a tool you can use to experiment with querying your data. Learn more about using this tool in the [Gatsby tutorial](https://www.gatsbyjs.org/tutorial/part-five/#introducing-graphiql).
 
+#### Running Tests
 
+While you are developing, we recommend watching the test suite. Open up another terminal in the site directory and run 
+```
+npm run test-watch
+```
 
-## Folder Structure
+## Project Structure
+Awesome Learning is powered by two repositories, this one and our [exercises repo](https://wayfair.github.io/awesome-learning-exercises/).
```

Comment:
I think you mean to link to `https://github.com/wayfair/awesome-learning-exercises` here.